### PR TITLE
Cleanup 201706

### PR DESCRIPTION
**Pull request by telmich**
Cleaning up some old stuff + email address. Review & merge, please

### CODE REVIEW

**Comment by telmich** [comment]
The massive file deletion are files in media/ that should never have been there

File: dynamicweb/settings/prod.py [hunk 1]
```diff
@@ -1,12 +1,8 @@
 from .base import *
 
-ADMINS = (
-    ('Nico Schottelius', 'nico.schottelius@ungleich.ch'),
-    ('Raul Ascencio', 'raul.ascencio@yandex.com'),
-    ('Web team', 'web-team@ungleich.ch')
+# List of people that get admin messages
+ADMINS = ( (x, x + "@ungleich.ch") for x in ["web-team"] )
```
Comment (Modulos) [inline dynamicweb/settings/prod.py]: Maybe we could use an env var instead

**Comment by telmich** [comment]
redoing against develop

File: dynamicweb/settings/prod.py (hunk 1 above)
Comment (telmich) [inline dynamicweb/settings/prod.py]: Might be a good idea to allow to define *additional* emails - I can fix that in a new PR later, I wanted to get rid of the plain mails first

**Pull request closed without merging**